cli/cmd/inject.go

```diff
@@ -103,6 +103,10 @@ sub-folders, or coming from stdin.`,
 		&options.disableIdentity, "disable-identity", options.disableIdentity,
 		"Disables resources from participating in TLS identity",
 	)
+	flags.BoolVar(
+		&options.disableTap, "disable-tap", options.disableTap,
+		"Disables resources from from being tapped",
+	)
```

Comment:
until the proxy supports it, should we do `flags.MarkHidden("disable-tap")` ? maybe reference a Github issue in a comment for the proxy change? or, since the control-plane is also filtering, maybe this is ok?

related: does this mean both the control-plane and the proxy will independently guard against tapping?

Reply:
As is, tapping gets disabled through the control plane API, but that doesn't disallow someone hitting directly the proxy. I think it makes sense to mark the flag hidden in the meantime.

Reply:
Created #2811 to track proxy-side work